### PR TITLE
Update goja

### PR DESCRIPTION
**Pull request by MStoykov**

### CODE REVIEW

File: js/modules/k6/ws/ws.go [hunk 1]
```diff
@@ -123,24 +123,19 @@ func (mi *WS) Exports() modules.Exports {
...
-	var callableV, paramsV goja.Value
-	switch len(args) {
-	case 2:
-		paramsV = args[0]
-		callableV = args[1]
-	case 1:
-		paramsV = goja.Undefined()
-		callableV = args[0]
-	default:
+	if callableV == nil {
```
Comment (codebien) [inline js/modules/k6/ws/ws.go:134]:
```suggestion
    // The params argument is optional
	if callableV == nil {
```
I'm not sure the next time I will be able to understand **why** we are doing it. So, I would prefer to maintain the comment explaining it.

**Review by codebien — CHANGES_REQUESTED** [review]
Just a small suggestion

**Comment by olegbespalov** [comment]
LGTM :+1:

**Review by olegbespalov — DISMISSED** [review]

File: js/modules/k6/ws/ws.go (hunk 1 above)
Comment (MStoykov) [reply js/modules/k6/ws/ws.go]: the reason for this changes was fixed upstream so I am dropping them

**Review by olegbespalov — APPROVED** [review]

**Review by codebien — APPROVED** [review]
LGTM

**Pull request merged**